pkg/controller/releasemanager/syncer.go

```diff
@@ -448,6 +448,7 @@ func (r *ResourceSyncer) prepareServiceMonitors() []*picchuv1alpha1.ServiceMonit
 		for _, i := range alertable {
 			if i.target() != nil {
 				sm = i.target().ServiceMonitors
+				break
```

Comment:
is this the “real fix”?

Reply:
No but I noticed this missing break when I looked at the other function. I'm pretty sure it's the nil check on `i.Target()` that is the "real fix"